test/integration/consul-container/test/gateways/gateway_endpoint_test.go

```diff
@@ -179,6 +179,25 @@ func checkHTTPRouteConfigEntry(t *testing.T, client *api.Client, routeName strin
 	}, time.Second*10, time.Second*1)
 }
 
+func checkHTTPRouteConfigEntryWithoutBindStatus(t *testing.T, client *api.Client, routeName string, namespace string) {
```

Comment:
I don't think this actually gets called in the tests, is that expected/is this going to help another PR?

Reply:
it doesn't here, it's used in an ENT PR for cross namespace tests, but figured it should go into OSS first so it's ported to ENT